extensions/amp-dailymotion/0.1/test/test-amp-dailymotion.js

```diff
@@ -76,6 +84,15 @@ describes.realWin(
       );
     });
 
+    it('renders already muted when autoplay is enabled', async () => {
+      const dailymotion = await getDailymotion('x2m8jpp', false, false, true);
+      const iframe = dailymotion.querySelector('iframe');
+      expect(iframe).to.not.be.null;
```

Comment:
Interesting that `.to.not.be.null` works. I've seen `not.to.be.null`. We have instances of both in our code base.

Reply:
My preference is to avoid the words that have no effect. In this case that would mean `.not.null`